src/java.base/share/classes/java/io/PrintStream.java

```diff
@@ -679,8 +679,6 @@ private void implWrite(byte[] buf, int off, int len) throws IOException {
      *
      * @see #writeBytes(byte[])
      * @see #write(byte[],int,int)
-     *
-     * @since 14
```

Comment:
The `@since 14` was added here as part of https://bugs.openjdk.org/browse/JDK-8187898. The CSR explains what the changes were https://bugs.openjdk.org/browse/JDK-8230625. As noted in that CSR's specification (and attached images), the change for this method in that issue ended up being just API clarification (through a `@apiNote`) and no other change. It continued to have the same signature including the throws clause that was previously available on this class through the super `FilterOutputStream#write(byte[])` method.

So removing this `@since 14` looks correct to me.